stable/admin/templates/statefulset.yaml

```diff
@@ -160,8 +160,39 @@ spec:
           mountPath: /etc/nuodb/keys/nuocmd.pem
           subPath: {{ .Values.admin.tlsClientPEM.key }}
         {{- end }}
+{{- if and .Values.insights.enabled .Values.admin.insights }}
```

Comment:
`.Values.admin.insights` is string. I was wondering how empty string will evaluate. It looks like an empty string is `false`.

Reply:
yes, empty is false, which is what I want.  If you want to disable for admin but not for engine you can set the admin.insights to "" and insights.enabled to true.